### PR TITLE
Added Stats for Stork Migrationschedule

### PR DESCRIPTION
**Pull request by gejain**
Added Stats for Stork Migrationschedule

**Which issue(s) this PR fixes** (optional)
Closes # https://portworx.atlassian.net/browse/PTX-22671

**Special notes for your reviewer**:
Getting stats like below -

image

### CODE REVIEW

**Review by kphalgun-px — APPROVED** [review]
LGTM

File: pkg/asyncdr/asyncdr.go [hunk 1]
```diff
@@ -295,10 +296,11 @@ func CreateSchedulePolicy(policyName string, interval int) (pol *storkapi.Schedu
 }
 
 // WaitForNumOfMigration waits for a certain number of migrations to complete.
-func WaitForNumOfMigration(schedName string, schedNamespace string, count int, miginterval int) (map[string]string, error) {
+func WaitForNumOfMigration(schedName string, schedNamespace string, count int, miginterval int) (map[string]string, []map[string]string, error) {
 	migInterval := time.Minute * time.Duration(miginterval)
 	migTimeout := time.Minute * time.Duration(count*miginterval)
 	expectedMigrations := make(map[string]string)
+	var migschedulestats []map[string]string
```
Comment (kphalgun-px) [inline pkg/asyncdr/asyncdr.go:303]: How about appending keys to a single map rather than creating a list of maps?

Comment (gejain) [reply pkg/asyncdr/asyncdr.go:303]: This wont work as they will be separate maps for seperate migrations. Multiple migrations created as a part of schedule and they disappears as next become successful, so as soon as one become successful we have to collect data in map for individual.

Comment (kphalgun-px) [reply pkg/asyncdr/asyncdr.go:303]: Oh okay. Thanks!

**Review by Rohit-PX — APPROVED** [review]
Please rebase and squash all commits.

**Pull request merged**